pkg/cli/image/manifest/dockercredentials/auth_resolver.go

```diff
@@ -50,12 +50,14 @@ func NewAuthResolver(authFilePath string) (*AuthResolver, error) {
 		if err != nil {
 			return nil, err
 		}
-		if pref, warn := image.GetRegistryAuthConfigPreference(); pref == image.DockerPreference {
+		// TODO: remove REGISTRY_AUTH_PREFERENCE env variable support and support only podman in 4.15
+		pref, warn := image.GetRegistryAuthConfigPreference()
+		if len(warn) > 0 {
+			fmt.Fprint(os.Stderr, warn)
+		}
+		if pref == image.DockerPreference {
 			config := defaultClientDockerConfig()
```

Comment:
this change is only done so we can print the warning if REGISTRY_AUTH_PREFERENCE is used also with the podman value